packages/lab/src/multiline-input/MultilineInput.css

```diff
@@ -0,0 +1,246 @@
+/* Style applied to the root element */
+.saltMultilineInput {
+  --multilineInput-borderColor: var(--salt-editable-borderColor);
+  --multilineInput-borderStyle: var(--salt-editable-borderStyle);
+  --multilineInput-outlineColor: var(--salt-focused-outlineColor);
+  --multilineInput-borderWidth: var(--salt-size-border);
```

Comment:
this var name is a bit confusing. Name suggests it's the wrapper border, but it's used only for the activationIndicator. Can we rename somehow?